containers/README.md

```diff
@@ -13,14 +13,14 @@ Note: the `docker push` part of the script assumes your Docker Hub creditionals
 (can be done via `docker login docker.io`, for example).
 
 ```shell
-export EESSI_PILOT_VERSION=2020.10
+export EESSI_PILOT_VERSION=2020.12
 ./docker_build_push.sh ${EESSI_PILOT_VERSION}
 ```
 
 ### Run (using Singularity)
 
 ```
-export EESSI_PILOT_VERSION=2020.10
+export EESSI_PILOT_VERSION=2020.12
```

Comment:
@bedroge I say we kick out the version part, it doesn't make much sense to have it there, since there's nothing version-specific in these containers...